packages/crawler/src/types.ts

```diff
@@ -0,0 +1,71 @@
+import { TagName } from 'shared/src';
```

Comment:
Should be imported from `@porsche-design-system/shared`

Reply:
Fixed